### PR TITLE
code generation for aether-4.0 models

### PR DESCRIPTION
**Pull request by sbconsulting**

### CODE REVIEW

File: modelplugin/aether-4.0.0/modelplugin/modelplugin.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+const modelversion = "4.0.0"
+const modulename = "aether.so.4.0.0"
+
+var ModelData = []*gnmi.ModelData{
+	{Name: "connectivity-service", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "enterprise", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+
+	{Name: "aether-types", Organization: "Open Networking Foundation", Version: "2021-09-10"},
+	{Name: "ap-list", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "application", Organization: "Open Networking Foundation", Version: "2021-06-02"},
```
Comment (SeanCondon) [inline modelplugin/aether-4.0.0/modelplugin/modelplugin.go:44]: `application` should be `2021-09-10`

File: modelplugin/aether-4.0.0/modelplugin/modelplugin.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	{Name: "device-group", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "ip-domain", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "site", Organization: "Open Networking Foundation", Version: "2021-09-10"},
+	{Name: "upf", Organization: "Open Networking Foundation", Version: "2021-06-02"},
+	{Name: "vcs", Organization: "Open Networking Foundation", Version: "2021-09-10"},
+	{Name: "template", Organization: "Open Networking Foundation", Version: "2021-09-10"},
```
Comment (SeanCondon) [inline modelplugin/aether-4.0.0/modelplugin/modelplugin.go:50]: `traffic-class` should be `2021-09-10` and `template` should be `2021-06-02`

File: modelplugin/aether-4.0.0/description.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+* `device-group`. A list of `Device-Group` objects that can participate in this `VCS`. Each
+  entry in the list contains both the reference to the `Device-Group` as well as an `enable`
+  field which may be used to temporarily remove access to the group.
+* `application`. A list of `Application` objects that are either allowed or denied for this
+  `VCS`. Each entry in the list contains both a reference to the `Application` as well as an
+  `allow` field which can be set to `true` to allow the application or `false` to deny it.
+* `template`. Reference to the `Template` that was used to initialize this `VCS`.
+* `upf`. Reference to the User Plane Function (`UPF`) that should be used to process packets
+  for this `VCS`. It's permitted for multiple `VCS` to share a single `UPF`.
+* `ap`. Reference to an Access Point List (`AP-List`) that lists the access points for this
```
Comment (SeanCondon) [inline modelplugin/aether-4.0.0/description.md:107]: I think the `ap-list` is gone

File: modelplugin/aether-4.0.0/modelplugin/modelplugin.go (hunk 2 above)
Comment (sbconsulting) [reply modelplugin/aether-4.0.0/modelplugin/modelplugin.go:50]: Template will need it's version updated in roc-helm-charts. I'll set it back to 2021-06-02 and put that on the to-do list.

**Pull request merged**